jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -279,6 +279,7 @@ void java_bytecode_convert_classt::convert(
   class_type.set_is_inner_class(c.is_inner_class);
   class_type.set_is_static_class(c.is_static_class);
   class_type.set_is_anonymous_class(c.is_anonymous_class);
+  class_type.set_containing_class(c.containing_class);
```

Comment:
The java bytecode docs uses the terminology `outer_class` rather than `containing_class` so I suggest using that